### ISSUE TITLE
Java8 and Dates

### ISSUE THREAD

**Issue by dcrissman** [lightblue-platform/lightblue-core#584]
Java8 changes the Date model. I am creating this issue to investigate and resolve any issues we might have with that.

**Comment by alechenninger** [comment]
Correct me if I'm wrong, but the existing java.util.Date should be unaffected. There is new API available in java.time which is basically a port of joda time.

**Comment by bserdar** [comment]
Correct. All we need to do is to support those new types. Backend still
operates with Date.
